VMWare/vmware-vcenter/ingest/parser.yml

```diff
@@ -0,0 +1,78 @@
+name: vmware-vcenter
+pipeline:
+  - name: parsed_event
+    external:
+      name: grok.match
+      properties:
+        input_field: "{{original.message}}"
+        output_field: message
+        pattern: '%{SESSION_TYPE_1}|%{SESSION_TYPE_2}|%{SESSION_TYPE_3}|%{SESSION_TYPE_4}|%{SESSION_TYPE_5}|%{SESSION_TYPE_6}|%{CONNECTIONS}|%{FAULT}|%{FAULT_TYPE_2}|%{HTTP_LOGS_1}|%{HTTP_LOGS_2}|%{OTHERS_EVENTS}|%{OTHERS_EVENTS_TYPE_2}|%{OTHERS_EVENTS_TYPE_3}|%{OTHERS_EVENTS_TYPE_5}|%{OTHERS_EVENTS_TYPE_4}'
```

Comment:
I think you should prefix the pattern with syslog header pattern, as optional, like in [this format](https://github.com/SEKOIA-IO/intake-formats/blob/9d31abb30f147b0aa4e2fcad07c5aa40b4848b1a/Cisco/cisco-secure-firewall/ingest/parser.yml#L9), and remove the syslog header pattern from subpatterns.